example/rnn/cudnn_lstm_bucketing.py

```diff
@@ -76,12 +78,15 @@ def train(args):
     if args.stack_rnn:
         stack = mx.rnn.SequentialRNNCell()
         for layer in range(args.num_layers):
+            dropout = 0.0
+            if layer < (args.num_layers - 1):
+                dropout = args.dropout
             stack.add(mx.rnn.FusedRNNCell(args.num_hidden, num_layers=1,
-                    mode='lstm', prefix='lstm_%d'%layer,
+                    mode='lstm', prefix='lstm_%d'%layer, dropout=dropout,
```

Comment:
@piiswrong It's only applied in the middle of the stack. There's no dropout applied to the input of the first layer, or the output of the last layer. So, if manual explicit stacking of fused cells is used (as [here](https://github.com/dmlc/mxnet/blob/master/example/rnn/cudnn_lstm_bucketing.py#L80:L87)), rather than the whole stack being built as a black box (as [here](https://github.com/dmlc/mxnet/blob/master/example/rnn/cudnn_lstm_bucketing.py#L89:L90)), no CUDNN dropout will actually be applied. I guess for manually stacked FusedLSTM, this implies the need to manually apply dropout in MxNet, rather than counting on CUDNN to do it.